Examples/Scripts/PreFreeSurferPipelineBatchNHP.sh

```diff
@@ -0,0 +1,342 @@
+        #StrucScannerPatientPosition=HFS                                       # scanner's patient position in the DICOM, HFS (default) or HFP or FFS or FFP
+        #T1wSampleSpacing=".00000710000000000000"                              # readout time in [sec] (optional for B0 distortion correction)
+        #T2wSampleSpacing=".00000210000000000000"                              # readout time in [sec] (optional for B0 distortion correction)
+        #StrucTopupNegative="SEField_1_AP"                                     # negative phase encoding directions (LR or AP) (optional for B0 distortion correction)
+        #StrucTopupPositive="SEField_1_PA"                                     # positive phase encoding directions (RL or PA) (optional for B0 distortion correction)
+        #StrucSEDwellTime=".00062999983620004258"                              # dwell time in [sec] for fMRI (optional for B0 distortion correction)
+        #StrucSEUnwarpDir="y"                                                  # phase encoding direction for topup SEField data (optional for B0 distortion correction)
+        #StrucUnwarpDir=z                                                      # B0 unwarp direction, z (FH) for sagittal scan typical for human, z- (HF) for coronal scans typical for NHP
+    fi
+  
+    if [[ $T1wSampleSpacing != "" && $T1wSampleSpacing != "NONE" && $T1wSampleSpacing != "None" && $T2wSampleSpacing != "" && $T2wSampleSpacing != "NONE" && $T2wSampleSpacing != "None" && $StrucSEUnwarpDir != "None" && $StrucSEUnwarpDir != "" ]] ; then
+        T1wSampleSpacing=$T1wSampleSpacing  # read from Seriesinfo.txt
+        T2wSampleSpacing=$T2wSampleSpacing  # read from Seriesinfo.txt
+        UnwarpDir="$StrucUnwarpDir"         # Read direction of structural scan. read from Seriesinfo.txt
+        if [[ -n $StrucTopupPositive && -n $StrucTopupNegative && -n $StrucSEDwellTime && "$StrucTopupPositive" != NONE ]] ; then
+            SpinEchoPhaseEncodeNegative=${StudyFolder}/${Subject}/RawData/$(echo $StrucTopupNegative | awk '{print $1}') # first SEfield will be used
+            SpinEchoPhaseEncodePositive=${StudyFolder}/${Subject}/RawData/$(echo $StrucTopupPositive | awk '{print $1}') # first SEfield will be used
+            if [[ -n $StrucTopupPositive2  && -n $StrucTopupNegative2 && "$StrucTopupPositive2" != NONE ]] ; then
+                SpinEchoPhaseEncodeNegative2=${StudyFolder}/${Subject}/RawData/$(echo $StrucTopupNegative2 | awk '{print $1}') # first SEfield will be used
+                SpinEchoPhaseEncodePositive2=${StudyFolder}/${Subject}/RawData/$(echo $StrucTopupPositive2 | awk '{print $1}') # first SEfield will be used
+            fi
+            SEEchoSpacing="$(echo $StrucSEDwellTime | awk '{print $1}')"    # dwell time for SE field, read from SEEchoSpacing in Seriesinfo.txt
+            SEUnwarpDir="$(echo $StrucSEUnwarpDir | awk '{print $1}')"  # read from PhaseEncodinglist in Seriesinfo.txt. x or y (minus or not does not matter)
```

Comment:
Again, making a new variable with a more ambiguous name and `awk`ing for the first item are strange design choices to me.